ansible_events_ui/api/project.py

```diff
@@ -31,12 +33,22 @@ async def list_projects(db: AsyncSession = Depends(get_db_session)):
     "/api/projects/",
     response_model=schemas.ProjectRead,
     operation_id="create_projects",
+    status_code=ACCEPTED,
 )
 async def create_project(
     project: schemas.ProjectCreate, db: AsyncSession = Depends(get_db_session)
 ):
     found_hash, tempdir = await clone_project(project.url, project.git_hash)
     project.git_hash = found_hash
+
+    query = sa.select(sa.exists).where(projects.c.name == project.name)
```

Comment:
The query is incorrect. Please refer to my example in a previous comment.

This query generates the following SQL code:
```
-- Wrong
SELECT EXISTS (SELECT *) AS anon_1 
FROM project 
WHERE project.name = :name_1
```

The correct one:

```
query = sa.select(sa.exists().where(projects.c.name == project.name))
```
which generates the following SQL:
```
-- Good
SELECT EXISTS (SELECT * 
FROM project 
WHERE project.name = :name_1) AS anon_1
```